third_party/CMakeLists.txt

```diff
@@ -93,3 +93,45 @@ if(VELOX_ENABLE_ARROW)
                                            ${ARROW_LIBDIR}/libparquet.a)
 
 endif()
+
+if(VELOX_ENABLE_QPL)
```

Comment:
we usually put the cmake install logic under "CMake/resolve_dependency_modules". Could you move it there, for consistency?

@assignUser could you help review the cmake stuff?

Reply:
Agreed please move this you can use the QAT PR as reference: https://github.com/facebookincubator/velox/pull/5861/files#diff-437ff3efa9f6a2646f2b27b7c2cb597c9364edc498875813c5e77705df360dc3

Reply:
Thanks @assignUser . I add inteliaa.cmake to CMake/resolve_dependency_modules just like QAT